content/en/blog/2022/instrument-kafka-clients/index.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Instrumenting Apache Kafka clients with OpenTelemetry
+linkTitle: Instrumenting Apache Kafka clients with OpenTelemetry
```

Comment:
Replace with "Instrumenting Apache Kafka clients" to make the short title even shorter, we did the same for many other blog posts, see on the left side menu here:

https://deploy-preview-1677--opentelemetry.netlify.app/blog/2022/instrument-kafka-clients/

Reply:
Makes sense. Done!